api/src/main/java/org/openmrs/module/mirebalais/apploader/CustomAppLoaderFactory.java

```diff
@@ -1904,6 +1907,27 @@ private void enableMexicoClinic() {
                 and(sessionLocationHasTag(LocationTags.CONSULT_NOTE_LOCATION))));
     }
 
+    private void enableSierraLeoneOutpatient() {
```

Comment:
If this (and the Mexico consult) could be considered primary care, it might make sense to just have the single "primaryCare" component and then switch on Country.HAITI, Country.MEXICO, etc...

(Or if "primaryCare" isn't the right term, but there's a better umbrella term for the notes we use In Haiti, Mexico and Sierra Leone, switch to that)

Hopefully eventually we will come up with a way to have different visit notes per country by folder configuration (or some other means) like we currently do for basic forms, and this would set us up nicely for that.